recipes/pygnome/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: f4263ea7e373414ea66ca0f630b705b07e0131258b2fafad03d369b0f5ef0ee6
+
+build:
+  skip: True  # [py<38]
+  missing_dso_whitelist:
```

Comment:
There's got to be something wrong here -- missing_dso_whitelist  should be only rarely needed in special circumstances !?!?!

Reply:
Yeah well, this entire configuration file was working just fine locally on my mac.  Warnings?  Yeah a few, but nothing that should have broken the entire thing.

Whatever is causing this has to do with the Azure instances they are running this on.  Tough situation huh?  Something that doesn't even fail locally, but fails catastrophically in the GitHub pipeline.

Reply:
I suspect it's about the cross-compiling infrastructure -- we're not doing that at all locally, and even if it's not being done here, the system is in place.

Which means we need to figure out "build" vs "host".

Reply:
you can ping the right group on the PR for help (can't find the list right now) -- and/or ask in the conda community forum:

https://conda.discourse.group/

Reply:
> I suspect it's about the cross-compiling infrastructure -- we're not doing that at all locally, and even if it's not being done here, the system is in place.
> 
> Which means we need to figure out "build" vs "host".

I am sure that is an important distinction.  However not even the osx64 target is building.  I would like us to first and foremost get that target working since it is what we are familiar with.